src/main/java/net/coderbot/iris/Iris.java

```diff
@@ -154,7 +155,11 @@ private static boolean loadExternalShaderpack(String name) {
 				.filter(path -> path.endsWith("shaders"))
 				.findFirst();
 		} catch (IOException e) {
-			logger.error("Error while finding shaderpack for zip directory {}", shaderpackPath);
+			if (e instanceof ZipException && e.getMessage().equals("zip END header not found")) {
```

Comment:
Why not just say the zip is corrupted on all instances of ZipException? There's a few other types of ZipException that can occur and they too seem to be caused by corrupted zip files. In this case, zip END header not found seems to indicate that the file was truncated.